dali/server/daserver.cpp

```diff
@@ -148,6 +148,8 @@ static bool populateWhiteListFromEnvironment(IWhiteListWriter &writer)
         return false;
     Owned<IRemoteConnection> conn = querySDS().connect("/Environment", 0, 0, INFINITE);
```

Comment:
What happens if no environment - will this fail / return null?

Reply:
yes, but there is always an /Environment node. It is ensured (ensurePTree) at dali start time.